frontend/src/index.tsx

```diff
@@ -236,7 +237,16 @@ const Store: Component = () => {
             <hr />
             <ul class="w-full flex flex-grow flex-col gap-1 p-2">
               <Show when={!(lastSerial() === 0)} fallback={
-                <p class="text-center unimportant">Loading store..</p>
+                <div class="text-center unimportant">
+                  <p >This Webxdc Store
+                    is not connected to a Bot
+                    to get apps from.
+                  </p>
+                  <p >To get a working Webxdc Store,
+                    please send "Hi" to
+                    appbot@testrun.org
```

Comment:
think it needs to say a bit more regarding the e-mail address 
```suggestion
                  <p >To get a working Webxdc store,
                    you need to send "Hi" to a store address and use the store app you get back.  
                    If nothing else you may try sending "Hi" to xstore@testrun.org , an experimental 
                    community deployment offering some curated apps. 
```